### DISCUSSION TITLE
Undo for Moving to Trash

### DISCUSSION THREAD

**Discussion by tlindgren** [mikekazakov/nimble-commander#452] [Q&A]
I just started using Nimble Commander and am really happy with it - thanks for building it!

One basic question: is there no undo for moving a file to the trash? I expected this would be a basic feature since it's it's what other file managers do but maybe I'm missing something

**Comment by mikekazakov** [comment]
Hi, thank you!
This feature is not implemented - there wasn't enough interest in it so far.
